### PR TITLE
Chore/chain in custom token

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: pkg/repo/guild_config_token/pg.go [hunk 1]
```diff
@@ -36,19 +36,6 @@ func (pg *pg) UpsertMany(configs []model.GuildConfigToken) error {
...
-		UpdateAll: true,
-	}).Create(&configs).Error
-
-	if err != nil {
-		tx.Rollback()
-		return err
-	}
-
-	return tx.Commit().Error
+func (pg *pg) CreateOne(record model.GuildConfigToken) error {
```
Comment (trkhoi) [inline pkg/repo/guild_config_token/pg.go:39]: create new function, not delete old one

File: pkg/repo/guild_config_token/store.go [hunk 2]
```diff
@@ -5,5 +5,5 @@ import "github.com/defipod/mochi/pkg/model"
 type Store interface {
 	GetByGuildID(guildID string) ([]model.GuildConfigToken, error)
 	UpsertMany(configs []model.GuildConfigToken) error
-	UpsertOne(configs model.GuildConfigToken) error
+	CreateOne(token model.GuildConfigToken) error
```
Comment (trkhoi) [inline pkg/repo/guild_config_token/store.go:8]: create new function, not delete old one

File: pkg/repo/token/pg.go [hunk 3]
```diff
@@ -34,20 +33,6 @@ func (pg *pg) GetDefaultTokens() ([]model.Token, error) {
...
-		UpdateAll: true,
-	}).Create(&token).Error
-
-	if err != nil {
-		tx.Rollback()
-		return err
-	}
-
-	return tx.Commit().Error
+func (pg *pg) CreateOne(record model.Token) error {
```
Comment (trkhoi) [inline pkg/repo/token/pg.go:36]: create new function, not delete old one

File: pkg/repo/token/store.go [hunk 4]
```diff
@@ -7,5 +7,5 @@ type Store interface {
 	GetAllSupported() ([]model.Token, error)
 	GetByAddress(address string, chainID int) (*model.Token, error)
 	GetDefaultTokens() ([]model.Token, error)
-	UpsertOne(token model.Token) error
+	CreateOne(token model.Token) error
```
Comment (trkhoi) [inline pkg/repo/token/store.go:10]: create new function, not delete old one

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]
